src/components/TexterRequest.jsx

```diff
@@ -122,6 +133,20 @@ class TexterRequest extends React.Component {
       );
     }
 
+    if (
+      !this.userCanRequest(this.props.data.currentUser.memberships) &&
+      settings.showDoNotAssignMessage
+    ) {
+      return (
+        <Paper>
+          <div style={{ padding: "20px" }}>
+            <h3>Not allowed to request</h3>
+            <p>{settings.doNotAssignMessage}</p>
```

Comment:
And for default here maybe: Your ability to request texts has been suspended. Please a contact a text team leader for more information.

(I think some texter somewhere might complain about having a "how are you?" at the end 😅

Reply:
Reviewed these with @kohlee and @ninabaldwin today and we feel good with this language!

Reply:
> (I think some texter somewhere might complain about having a "how are you?" at the end 😅

Haha. That was me testing editing the message 😅